--- semantik-0.5.8/.waf-1.2.0-1940321410/wafadmin/Tools/qt4.py	2007-09-19 03:05:21.000000000 +0200
+++ semantik-0.5.8/.waf-1.2.0-1940321410/wafadmin/Tools/qt4.py-new	2007-10-19 19:14:58.521467918 +0200
@@ -212,7 +212,7 @@
 	if not qtdir:
 		try:
 			path = os.environ['PATH'].split(':')
-			for qmk in ['qmake-qt4', 'qmake4', 'qmake']:
+			for qmk in ['qmake-qt4', 'qmake4', 'qt4-qmake']:
 				qmake = conf.find_program(qmk, path)
 				if qmake:
 					version = os.popen(qmake+" -query QT_VERSION").read().strip().split('.')
